content/services/speaking.rst

```diff
@@ -19,7 +19,7 @@ summary: |
   site performance, and accessibility.
 
   .. _writing: /blog/
-  .. _improve design systems: /talks/agile-systems-workshop/
+  .. _improve design systems: /talks/agile-systems/
```

Comment:
@mirisuzanne This link was broken -- but it looks like the page is missing an image?

![Screenshot 2020-03-02 11 14 25](https://user-images.githubusercontent.com/552316/75694859-3186c180-5c77-11ea-92e0-1c79fb95248b.png)

Reply:
@jgerigmeyer fixed that header, but also redirected the link to our workshop page.